setup.cfg

```diff
@@ -4,3 +4,9 @@ description-file = README.md
 [tool:pytest]
 filterwarnings =
     ignore:the matrix subclass:PendingDeprecationWarning
+markers=
+  server: anything that connects to a server
+  upload: anything that uploads to a server
+  production: any interaction with the production server
+  cache: anything that interacts with the (test) cache
+
```

Comment:
Just going from this for documentation purposes: this would mean that if a test is marked with `server` but not with `production`, we know the test uses a test server?

Reply:
Yes, but since that's default I figured it wouldn't need a marker.